Cargo.Bazel.lock

```diff
@@ -1,5 +1,5 @@
 {
-  "checksum": "d7b1759c5d28cbeb3fff8b9da5ca81261aa8c593891b9eff7d3c554d07b4c8dd",
+  "checksum": "8c7bf9445e9dc6b2654902f1a47db51d487887849c7db5b8cb0e4e4f3ab30945",
```

Comment:
Just out of curiosity, why did this change?

Reply:
I guess because `iceoryx2-bb-derive-macros` was added to `Cargo.toml` or because a different bazel version was used on macOS?

Reply:
This change was developed on `ubuntu` 24.04. I didn't pay close attention to why it changed but also assumed it was because of changes in `iceoryx2-bb-elementary` and `iceoryx2-bb-derive-macros`.